puma/tests/test_histogram.py

```diff
@@ -144,6 +160,20 @@ def test_multiple_references_flavour(self):
         with self.assertRaises(ValueError):
             hist_plot.plot_ratios()
 
+    def test_two_ratio_panels(self):
+        """Tests if error is raised if more than 1 ratio panel is requested."""
+        with self.assertRaises(ValueError):
+            HistogramPlot(n_ratio_panels=2)
+
+    def test_multiple_references_wrong_flavour(self):
+        """Tests if warning is raised with wrong flavour."""
+        hist_plot = HistogramPlot(n_ratio_panels=1)
+        dummy_array = np.array([1, 1, 2, 3, 2, 3])
+        hist_plot.add(Histogram(dummy_array, flavour="dummy"), reference=True)
+        hist_plot.add(Histogram(dummy_array, flavour="dummy"))
+        hist_plot.plot()
+        hist_plot.plot_ratios()
```

Comment:
Maybe I'm missing something, but I think here we should test for the output of the logger.

We could use something like [this](https://testfixtures.readthedocs.io/en/latest/logging.html).

Reply:
changed it to capture logging